src/components/squareComponent/squareComponent.js

```diff
@@ -0,0 +1,16 @@
+import React, { Component } from 'react';
+import './squareComp.css'
+
+class Square extends Component {
+
+    render() {
+        return (
+            <div>
+                <a href="/"><div className="game-spaces" id="space"></div></a>
```

Comment:
Why is this a `<a>` tag? Could this just be a an unstyled `button` or just a `div`?

Reply:
Yea I was just thinking on this as I'm going over my code.  This isn't how I should do it. I'm going to create an onClick function that will log to the console the space has been clicked. Is that okay?

Reply:
A tag is removed, I've created a function that now handles the click event